pkg/installer/installer.go

```diff
@@ -0,0 +1,551 @@
+								},
+								"metadata": extensionsv1.JSONSchemaProps{
+									Type: "object",
+								},
+								"status": extensionsv1.JSONSchemaProps{
+									Type:        "object",
+									Description: "TableStatus defines the observed state of Table",
+									Properties: map[string]extensionsv1.JSONSchemaProps{
+										"plans": extensionsv1.JSONSchemaProps{
+											Type: "array",
+											Items: &extensionsv1.JSONSchemaPropsOrArray{
+												Schema: &extensionsv1.JSONSchemaProps{
+													Type: "object",
+													Properties: map[string]extensionsv1.JSONSchemaProps{
+														"approvedAt": extensionsv1.JSONSchemaProps{
+															Type:   "integer",
+															Format: "int64",
+														},
+														"ddl": extensionsv1.JSONSchemaProps{
+															Type: "string",
+														},
+														"executedAt": extensionsv1.JSONSchemaProps{
+															Type:   "integer",
+															Format: "int64",
+														},
+														"invalidatedAt": extensionsv1.JSONSchemaProps{
+															Type:        "integer",
+															Format:      "int64",
+															Description: "InvalidatedAt is the unix nano timestamp when this plan was determined to be invalid or outdated",
+														},
+														"name": extensionsv1.JSONSchemaProps{
+															Type: "string",
+														},
+														"plannedAt": extensionsv1.JSONSchemaProps{
+															Type:        "integer",
+															Format:      "int64",
+															Description: "PlannedAt is the unix nano timestamp when the plan was generated",
+														},
+														"rejectedAt": extensionsv1.JSONSchemaProps{
+															Type:   "integer",
+															Format: "int64",
+														},
+													},
+												},
+											},
+										},
+									},
+								},
+								"spec": extensionsv1.JSONSchemaProps{
+									Description: "TableSpec defines the desired state of Table",
+									Type:        "object",
+									Required:    []string{"database", "name", "schema"},
+									Properties: map[string]extensionsv1.JSONSchemaProps{
+										"database": extensionsv1.JSONSchemaProps{
+											Type: "string",
+										},
+										"name": extensionsv1.JSONSchemaProps{
+											Type: "string",
+										},
+										"requires": extensionsv1.JSONSchemaProps{
+											Type: "array",
+											Items: &extensionsv1.JSONSchemaPropsOrArray{
+												Schema: &extensionsv1.JSONSchemaProps{
+													Type: "string",
+												},
+											},
+										},
+										"schema": extensionsv1.JSONSchemaProps{
+											Type: "object",
+											Properties: map[string]extensionsv1.JSONSchemaProps{
+												"mysql": extensionsv1.JSONSchemaProps{
```

Comment:
could you set this to a var if its duplicated under mysql and postgres?

Reply:
👍